### PR TITLE
[Fix] Made Admin User Table Page code into MVC standard & Dashboard Graphs French Translation/View Issues

### PR DESCRIPTION
**Pull request by SukhsimranpreetSekhon**
- Made component userTable folder with controller (UserTable.jsx) and view (UserTableView.jsx).
- Fixed fench translation issue on dashboard and user table page for Admin.
- Cleaned up view code in DashboardGraphsView.jsx file.

### CODE REVIEW

File: services/frontend-v3/src/components/userTable/UserTable.jsx [hunk 1]
```diff
@@ -0,0 +1,420 @@
+import React, { Component } from "react";
+import UserTableView from "./UserTableView";
+import { LinkOutlined, SearchOutlined } from "@ant-design/icons";
+import { Select, Button, Input, Skeleton } from "antd";
```
Comment (Rizvida1) [inline services/frontend-v3/src/components/userTable/UserTable.jsx]: There shouldn't be antd in controllers

**Review by Rizvida1 — CHANGES_REQUESTED** [review]

**Comment by dynamic11** [comment]
what is the status of this pull request?

File: services/frontend-v3/src/components/userTable/UserTable.jsx (hunk 1 above)
Comment (SukhsimranpreetSekhon) [reply services/frontend-v3/src/components/userTable/UserTable.jsx]: Fixed, only have Skeleton for loading data.

**Comment by SukhsimranpreetSekhon** [comment]
> what is the status of this pull request?

**Review by Rizvida1 — APPROVED** [review]

**Pull request merged**